src/components/molecules/storage/FileListItem.tsx

```diff
@@ -44,6 +45,21 @@ const FileListItem = ({ file, arweave, arweaveWallet }: FileListItemProps) => {
     setArweaveTransaction(transaction);
   };
 
+  const pin = async (node: any, cid: string) => {
+    const myHeaders = new Headers();
+    myHeaders.append('Authorization', 'Basic aXBmcy1jbHVzdGVyOmZqZHNha2ZsZHNhZjk4ZndqZTczMg==');
+
+    fetch('https://ipfs-cluster.akropolis.turbinekreuzberg.io/pins/' + cid, {
```

Comment:
Since that url also is a somewhat public / configurable service it shouldn't go into code here as well (the public ipfs peers might be a different thing since they're far from "secret" and quite ephemeral)